runtime/README.md

```diff
@@ -481,6 +481,11 @@ If a DPE command were to activate a new context such that the total number of
 active contexts in a privilege level is above its active context limit, the 
 InvokeDpe command should fail. 
 
+Further, it is not allowed for PL1 to call DeriveChild with the intent of 
+changing locality to PL0's locality, since this would increase the number 
+of active contexts in PL0's locality, and hence allow PL1 to impersonate
```

Comment:
nit: I think it would just exhaust the contexts, not let PL1 impersonate PL0

```suggestion
of active contexts in PL0's locality, and hence allow PL1 to DOS
```

Reply:
Done